data-prepper-plugins/kafka-plugins/src/test/resources/kafka-pipeline-msk-sasl-plain.yaml

```diff
@@ -0,0 +1,20 @@
+log-pipeline :
+  source:
+     kafka:
+        encryption:
+          type: "SSL"
+        authentication:
+          sasl:
+            plain:
+              username: username
```

Comment:
Use some values that are more unique: `test_sasl_username` and `test_sasl_password` would work.